package.json

```diff
@@ -7,16 +7,16 @@
     "build:js": "esy b dune build ./test/bin/test_runner.bc.js",
     "test:js": "node bin/test-jsoo.js",
     "test:native": "esy b dune runtest",
-    "test": "npm run test:js && npm run test:native"
+    "test": "npm run build:js && npm run test:js && npm run test:native"
   },
   "esy": {
     "build": ["refmterr dune build -p reglm"],
     "buildsInSource": "_build"
   },
   "resolutions": {
     "@esy-ocaml/esy-installer": "0.0.1",
-    "@opam/cmdliner": "1.0.2",
-    "@opam/dune": "1.7.3"
+    "@opam/cmdliner": "1.0.4",
+    "@opam/camomile": "1.0.1"
```

Comment:
cmdliner 1.0.2 or 1.0.4 compile on windows but 1.0.3 don't same for camomile 1.0.2